agent/grpc-external/services/dataplane/get_envoy_bootstrap_params.go

```diff
@@ -73,7 +74,24 @@ func (s *Server) GetEnvoyBootstrapParams(ctx context.Context, req *pbdataplane.G
 		NodeId:      string(svc.ID),
 	}
 
-	bootstrapConfig, err := structpb.NewStruct(svc.ServiceProxy.Config)
+	// This is awkward because it's designed for different requests, but
+	// this fakes the ServiceSpecificRequest so that we can reuse code.
+	_, ns, err := s.MergeNodeServiceWithCentralConfig(
+		nil,
```

Comment:
I pass a nil here for the `memdb.WatchSet`. Is that fine? 

We also do make multiple reads from the store: one for the `store.ServiceNode` and then at least one here to read the config entries, so I was wondering if those should maybe be batched into a transaction or such?

Reply:
Yep, calling `Add` on a nil watch set is a no-op, so it's fine.

As for doing all the queries in a single transaction, I guess the main benefit would be [isolation](https://en.wikipedia.org/wiki/Isolation_(database_systems)) - that is, you'd never get a mix of config entries from different versions of the database (before and after a write).

Which is an important enhancement, but I agree with @riddhi89 isn't critical for right now.